src/services/localesServices/locales/fr.json

```diff
@@ -1,7 +1,9 @@
 {
-    "Welcome to devRepublic Barefoot Nomad API": "Bienvenue au devRepublic Barefoot Nomad API",
-    "register": "enregistrer",
-    "signup": "enregistrer",
-    "login": "connecter",
-    "logout": "déconnecter"
-}
+	"Welcome to devRepublic Barefoot Nomad API": "Bienvenue au devRepublic Barefoot Nomad API",
+	"register": "enregistrer",
+	"signup": "enregistrer",
+	"login": "connecter",
+	"logout": "déconnecter",
+	"User is successfully logged out": "l'utilisateur s'est déconnecté avec succès",
+	"Login first or create an account if you do not have one": "Connectez-vous d'abord ou créez un compte si vous n'en avez pas"
```

Comment:
one is in english

Reply:
I don't get what you mean exactly ?

Reply:
I didn't use that last error. I am going to remove it.

Reply:
@bbaime98 you should include the message you get when logged in

Reply:
And all possible error and success messages here too for both languages